.travis.yml

```diff
@@ -125,18 +125,23 @@ jobs:
       script: ./travisci/perl-linter_harness.sh
 
     - language: python
-      python: 3.8
+      python: "3.10"
+      dist: bionic
       name: "Python unit tests on the minimum version"
       install:
-        - pip install -r requirements.txt
+        - pip install --upgrade pip
+        - pip install codecov
```

Comment:
This could perhaps be dropped as coverage is being checked on the latest version.
```suggestion
```